### DISCUSSION TITLE
Super Thanks

### DISCUSSION THREAD

**Discussion by pikapictured** [ReVanced/revanced-discussions#991] [Question] [answered]
**I don't know where to post this, I will post this here for now.**

YouTube ReVanced is one of the best apps that saved me from the hell of ads, bringing back the dislikes, and has SponsorBlock support.

The only problem I have is that Super Thanks doesn't work, I don't know if it's because of the MicroG Vanced or if I used the ReVanced Patcher wrong. Can someone please help me? Thanks. 😊

**Comment by oSumAtrIX** [comment]
You can open an issue [here](https://github.com/revanced/revanced-patches/issues/new/choose).

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
Making payments in patched YouTube doesn't work, due to limitations of MicroG.

You'll have to temporarily use unpatched YT to make any payments.
